arm/examples/client.go

```diff
@@ -50,7 +49,7 @@ func main() {
 	spt, err := helpers.NewServicePrincipalTokenFromCredentials(c, azure.AzureResourceManagerScope)
 	if err != nil {
 		fmt.Println(err)
```

Comment:
Ideally you should never call fmt.println in this pkg for errors as it's a very short program and exits on every error. Just log.fatal